Auth0Tests/AuthenticationSpec.swift

```diff
@@ -808,6 +809,143 @@ class AuthenticationSpec: QuickSpec {
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: "audience", scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should not include audience if it is nil") {
+                    stub(condition: isToken(Domain) && hasNoneOf(["audience"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should not include audience by default") {
+                    stub(condition: isToken(Domain) && hasNoneOf(["audience"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should include scope if it is not nil") {
+                    stub(condition: isToken(Domain) && hasAtLeast(["scope": "scope"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, scope: "scope", parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should not include scope if it is nil") {
+                    stub(condition: isToken(Domain) && hasNoneOf(["scope"])) { _ in
+                        return authResponse(accessToken: AccessToken)
+                    }
+                    waitUntil(timeout: Timeout) { done in
+                        auth.login(email: SupportAtAuth0, code: OTP, audience: nil, scope: nil, parameters: [:]).start { result in
+                            expect(result).to(beSuccessful())
+                            done()
+                        }
+                    }
+                }
+                
+                it("should use 'openid' as the default scope") {
```

Comment:
🤔 I thought there was no default scope set above?

Reply:
Answered in a comment above.